flask_common/crypto.py

```diff
@@ -3,6 +3,10 @@
 from Crypto.Util import Counter
 import hashlib
 import hmac
+import sys
+
+if sys.version_info > (3,):
```

Comment:
Probably should just remove this for now and replace with a `TODO` that we need to make this py3 compatible